proposals/function-pointers.md

```diff
@@ -235,6 +235,23 @@ void* v = &CloseHandle;
 func* cdecl bool(IntPtr) f1 = (func* cdecl bool(IntPtr))v;
 ```
 
+### Extensible set of unmanaged calling conventions
+
+The set of unmanaged calling conventions supported by the current ECMA-335 encodings is outdated. We have seen requests to add support
+for more unmanaged calling conventions, for example:
+
+- [vectorcall](https://docs.microsoft.com/en-us/cpp/cpp/vectorcall) https://github.com/dotnet/coreclr/issues/12120
+- StdCall with explicit this https://github.com/dotnet/coreclr/pull/23974#issuecomment-482991750
+
+The design of this feature should allow extending the set of unmanaged calling conventions as needed in future. The problems include
+limitted space for encoding calling conventions (12 out of 16 values are taken in `IMAGE_CEE_CS_CALLCONV_MASK`) and number of places
+that need to be touched in order to add a new calling convention. A potential solution is to introduce a new encoding that represents
+the calling convention using `System.Runtime.InteropServices.CallingConvention` enum.
```

Comment:
Please embed the link to the official documentation for the enum - https://docs.microsoft.com/dotnet/api/system.runtime.interopservices.callingconvention